middleware/cached.js

```diff
@@ -37,13 +37,15 @@ function sMaxAge(sMaxAgeValue) {
 
 /**
  * csrfProtection
- * Apply csrf protection and no-cache at the same time
+ * Apply csrf protection and no-store at the same time
  */
-const csrfProtection = [csurf(), noCache];
+const csrfProtection = [csurf(), noStore];
 
 module.exports = {
     defaultMaxAge,
-    noCache,
+    // @TODO: Replace all instances of noCache with noStore
```

Comment:
`noCache` is a misleading name as it's setting the `no-store` value (`no-cache` is a semantically different thing). Added a TODO here to remind us to renamed the other uses.